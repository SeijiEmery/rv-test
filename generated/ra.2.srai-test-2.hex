0000000    93  d0  10  40  73  00  10  00                                
0000010
