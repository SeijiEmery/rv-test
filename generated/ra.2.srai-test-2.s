srai x1, x1, 1
ebreak
