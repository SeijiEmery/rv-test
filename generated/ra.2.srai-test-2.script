load /x 0 generated/ra.2.srai-test-2.hex
writereg 1 9223372036854775808
run 0 100000
readreg 1