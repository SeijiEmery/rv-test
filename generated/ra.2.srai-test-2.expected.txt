R1 = 13835058055282163712
